src/pages/FactPage.tsx

```diff
@@ -0,0 +1,20 @@
+import "./FactPage.scss";
+import { getFacts } from "../data/factData";
+import FactCard from "../components/FactCard";
+
+const FactPage = () => {
+  return (
+    <div className="facts-wrapper">
```

Comment:
Do you need this wrapper? I see you style it to be flex and display in a column, but isn't that the default behaviour of block elements?

Also, I don't think the `<h1>` on the next line should be inside this element

Reply:
facts-wrapper removed. 
h1 now top of page component